settings/deno.vim

```diff
@@ -13,7 +13,11 @@ augroup vim_lsp_settings_deno
       \     'implementations': v:true,
       \     'references': v:true,
       \     'referencesAllFunctions': v:true,
+      \     'test': v:true,
+      \     'testArgs': ['--allow-all'],
       \   },
+      \   'config': empty(lsp#utils#find_nearest_parent_file(lsp#utils#get_buffer_path(), 'tsconfig.json')) ? v:null : lsp#utils#find_nearest_parent_file(lsp#utils#get_buffer_path(), 'tsconfig.json'),
```

Comment:
[memo]
If tsconfig.json is exists at root directory, send to deno otherwise set null